### PR TITLE
ISSUE-12: Implemented configuration file support.

### PR DESCRIPTION
**Pull request by cbolgiano**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -3,11 +3,15 @@ package main
 import (
 	"context"
 	"crypto/x509"
+	"encoding/json"
 	"flag"
+	"fmt"
```
Comment (ebastos) [inline main.go]: This import is not required

File: main.go [hunk 2]
```diff
@@ -3,11 +3,15 @@ package main
...
+	"io/ioutil"
 	"log"
 	"log/syslog"
 	"os"
 	"os/user"
+	"strconv"
```
Comment (ebastos) [inline main.go]: This one neither

File: main.go [hunk 3]
```diff
@@ -22,8 +26,34 @@ const (
 	timeout = 300 * time.Millisecond
 )
 
-func connect(address string) (*grpc.ClientConn, error) {
+// Config
+type Config struct {
+	Enabled   boolean `json:"enabled"`
```
Comment (ebastos) [inline main.go]: Should be "bool" instead of "boolean"

File: main.go [hunk 4]
```diff
@@ -59,6 +89,13 @@ func getinformation(argsWithoutProg []string, commandExitCode int64) spb.Command
 }
 
 func main() {
+        config := config()
+
+        if !strconv.ParseBool(config.Enabled) {
```
Comment (ebastos) [inline main.go]: This one can be simply `if !config.Enabled {`

File: main.go [hunk 5]
```diff
@@ -59,6 +89,13 @@ func getinformation(argsWithoutProg []string, commandExitCode int64) spb.Command
...
+                log.Fatal("shell-history is not enabled.")
```
Comment (ebastos) [inline main.go]: Since the user intentionally disabled the logging, we should not log a problem here.
Just `return` and it should be fine.

**Comment by cbolgiano** [comment]
Updated PR with suggested changed.  Please let me know if there is anything else I can do.  Thank you for the good feedback.

**Comment by ebastos** [comment]
Thanks, @cbolgiano 

I just did a minor change. This is merge now.

**Pull request merged**